portfolio/src/main/java/com/google/sps/servlets/ServeURLServlet.java

```diff
@@ -0,0 +1,22 @@
+import java.io.IOException;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.annotation.WebServlet;
+
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+
+@WebServlet("/serve")
```

Comment:
Can you add class documentation describing what this servlet does?

Reply:
Done